test/built-ins/Proxy/defineProperty/trap-is-undefined.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.6
+description: >
+    Return target.[[DefineOwnProperty]](P, Desc) if trap is undefined.
+info: >
+    [[DefineOwnProperty]] (P, Desc)
+
+    ...
+    8. If trap is undefined, then
+        a. Return target.[[DefineOwnProperty]](P, Desc).
+    ...
+---*/
+
+var target = {};
+var p = new Proxy(target, {});
+
+Object.defineProperty(p, "attr", {
+    configurable: true,
+    enumerable: true,
+    writable: true,
+    value: 1
+});
+
+var desc = Object.getOwnPropertyDescriptor(target, "attr");
+assert(desc.enumerable);
```

Comment:
Check out `harness/propertyHelper.js`--you can use the `includes` frontmatter tag to define that for tests like this.